package.json

```diff
@@ -23,8 +23,8 @@
     "quests-local": "API_HOST='http://localhost:8081' npm run quests & npm run api"
   },
   "dependencies": {
-    "@material-ui/core": "^1.2.1",
-    "@material-ui/icons": "^1.1.0",
+    "@material-ui/core": "^3.5.1",
```

Comment:
With such a large version increase in such a core package, can you run through our beta test checklist to make sure the rest of the app + quest creator UI still looks and behaves reasonably? https://docs.google.com/document/d/1HkHct0-AEMaHpkpU0uqSi6mhj2GJId78bXc23mo-2qg/edit

Reply:
Can you please grant access to `nimish@pesto.tech` for the above doc?

Also, I have bumped this version just for figuring out the travis issue with tooltip. Probably I will change this to before as we are not requiring tooltip.